### PR TITLE
Open search in new tab

### PR DESCRIPTION
**Pull request by asadiqbal08**
Task# [834](https://github.com/mitodl/mitxpro/issues/834)

@pdpinch Do you mean that to open that search results in new tab ? 

Screen Shot 2019-07-15 at 1 40 16 PM

Preview Theme [HERE](https://xpro.zendesk.com/theming/theme/138356f4-1254-4348-9efe-7dd96fe130df)

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@asadiqbal08 link provided in description of PR leads me to 404 page.
Screen Shot 2019-07-15 at 2 28 45 PM

File: script.js [hunk 1]
```diff
@@ -168,10 +168,16 @@ document.addEventListener('DOMContentLoaded', function() {
 });
 
 $( document ).ready(function() {
+
   $('a.submit-a-request').each(function () {
```
Comment (tasawernawaz) [inline script.js]: didn't get why we are looping here?

File: script.js [hunk 2]
```diff
@@ -168,10 +168,16 @@ document.addEventListener('DOMContentLoaded', function() {
...
-    if (location.pathname.indexOf('/requests/new') > -1) {
+    if (location.pathname.endsWith('/requests/new')) {
       $(this).hide();
       $('.contact-block').hide();
+
+      $("form.search").submit(function() {
+        $("form.search").attr('target', '_blank');
+        return true;
```
Comment (tasawernawaz) [inline script.js]: what is the use case of returning `ture` here?

**Review by tasawernawaz — COMMENTED** [review]
Honestly I don't know much about zendesk theme, have couple of questions.

File: script.js (hunk 2 above)
Comment (asadiqbal08) [reply script.js]: typo. not necessary

File: script.js (hunk 1 above)
Comment (asadiqbal08) [reply script.js]: hiding all `submit-a-request` links over the page but it is old implementation.

**Review by ahmed-arbisoft — APPROVED** [review]
Looks good as far as the `target="_blank"` is concerned. Not sure about the anchor hiding, don't have much background info on that.

**Comment by asadiqbal08** [comment]
No worry about it, It was the part of [594](https://github.com/mitodl/mitxpro/issues/594) before

**Comment by asadiqbal08** [comment]
@pdpinch a question for you. please take a look over it and let me should I go with merge ?

**Comment by pdpinch** [comment]
I didn't realize this has been waiting on me since 2019! 😳

Please go ahead and merge it.

**Pull request merged**